src/sentry/utils/outcomes.py

```diff
@@ -120,6 +123,18 @@ def track_outcome(
         ),
     )
 
+    if timestamp.replace(tzinfo=now.tzinfo) - now > LATE_OUTCOME_THRESHOLD:
```

Comment:
For a late outcome, is `timestamp` expected to be in the past, or future?
If past, then should it be:
```py
now - timestamp.replace(tzinfo=now.tzinfo > LATE_OUTCOME_THRESHOLD:
```

Reply:
oop yeah i switched that, lemme fix that, thanks!